libraries/stdlib/test/text/StringBuilderTest.kt

```diff
@@ -5,6 +5,12 @@ import org.junit.Test as test
 
 class StringBuilderTest {
 
+    @test fun stringBuildWithInitialCapacity() {
+        val s = buildString(123) {
+            assertEquals(123, capacity())
```

Comment:
This test won't compile in JS because there is no `StringBuilder.capacity` there.
To make this test JVM-only, move it to `StringBuilderJVMTest.kt`